packages/components/psammead-section-label/src/index.stories.jsx

```diff
@@ -1,57 +1,50 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { boolean, withKnobs } from '@storybook/addon-knobs';
-import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
 import notes from '../README.md';
 import SectionLabel from './index';
 
 storiesOf('Components|SectionLabel', module)
   .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
   .add(
     'default',
-    inputProvider({
-      slots: [{ name: 'title', defaultText: 'Most Read' }],
-      // eslint-disable-next-line react/prop-types
-      componentFunction: ({ slotTexts: [title], script, dir, service }) => (
-        <SectionLabel
-          script={script}
-          dir={dir}
-          bar={boolean('show bar?', true)}
-          visuallyHidden={boolean(
-            'visually hide component for all breakpoints?',
-            false,
-          )}
-          labelId="example-section-label"
-          service={service}
-        >
-          {title}
-        </SectionLabel>
-      ),
-    }),
+    ({ text, script, dir, service }) => (
+      <SectionLabel
+        script={script}
+        dir={dir}
+        bar={boolean('show bar?', true)}
+        visuallyHidden={boolean(
+          'visually hide component for all breakpoints?',
+          false,
+        )}
+        labelId="example-section-label"
+        service={service}
+      >
+        {service === 'news' ? 'Most read' : text}
```

Comment:
```suggestion
        {service === 'news' ? 'Most Read' : text}
```

this was flagged up as a difference on chromatic: https://www.chromaticqa.com/snapshot?appId=5d28eb5ee163f6002046d6fb&id=5da87bc6d0b479002050de4d

Reply:
spoke on slack, Jonathan is looking into the chromatic changes